k8s/arroyo/values.yaml

```diff
@@ -15,7 +15,7 @@ api:
   image:
     repository: ghcr.io/arroyosystems/arroyo-services
     pullPolicy: IfNotPresent
-    tag: "0.8.0"
+    tag: "tip"
```

Comment:
So that users who run the helm chart from source get 0.9.0-dev (i.e., the current dev version) rather than 0.8. Basically, aligns the helm chart in master with what you'd get building master directly. (We can't use 0.9, because that tag won't exist until we publish.)

Users running the helm chart from the published version at https://artifacthub.io/packages/helm/arroyo/arroyo will still get 0.8 until we release 0.9.